DEMOQA/cypress/fixtures/testData.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "firstName": "Phuong",
```

Comment:
Nice that you use fixture file, but you should store all test data in fixture file, I see you still hard code some test data like DoB, file upload path,...